### DISCUSSION TITLE
黑鷹直升機無人化再進一步，首度成功執行救援任務

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS3073-C/Python-Community-GS3073-C#553] [General]
美國陸軍、DARPA 與賽考斯基公司今年合作進行黑鷹直升機無人化測試，繼先前完成基本飛行和運貨任務測試後，近日更進一步完成緊急救援示範任務，朝實用範圍邁進。
這項測試於 10 月中在亞利桑那州的尤馬實驗場（Yuma Proving Ground）舉行，屬於陸軍大型「匯集計畫」（Project Convergence 2022）的一部分，匯集計畫是陸軍參與五角大廈「聯合全領域指揮控制」（JADC2）的計畫名稱，與英國和澳洲陸軍合作測試超過 300 項新型科技。
https://technews.tw/2022/11/08/ai-blackhawk-succefully-tested-in-rescue-mission/

**Comment by williamyen0623** [comment]
無人操作的武器被發明，就代表著他們不用犧牲軍人，就能輕鬆發起戰爭？

**Comment by limitx0** [comment]
直升機能無人駕駛，而且機艙內還有醫療系統
用在山難救援上可以有很大的幫助

**Comment by boson13579** [comment]
在天災發生時，直升機無人化更能在減少人員損失的前提下救出受難者。
